src/styles/base/_typography-mixins.scss

```diff
@@ -81,12 +81,12 @@
 
 @mixin text-sm {
 	font-size: .888888889rem;
-	line-height: $base-line-height;
+	line-height: $base-line-height-sm;
 }
 
 @mixin text-xs {
 	font-size: .777777778rem;
-	line-height: $base-line-height;
+	line-height: .714285714 * $base-line-height;
```

Comment:
Yeah good call actually. Line heights should probably be relative to the font size of the text and not the base. We should look to fix this elsewhere in future, but I'll update this for now.